sdk/storage/AzureStorageBlob/AzureStorageBlob.xcodeproj/project.pbxproj

```diff
@@ -422,6 +454,23 @@
 			shellPath = /bin/sh;
 			shellScript = "BUILD_PATH=$(pwd)\nROOT_PATH=$(pwd | rev | cut -d'/' -f4- | rev) \n$ROOT_PATH/Scripts/swiftlint.sh $BUILD_PATH\n";
 		};
+		8828E044AC1241DB30888F92 /* [CP] Embed Pods Frameworks */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+				"${PODS_ROOT}/Target Support Files/Pods-AzureStorageBlobTests/Pods-AzureStorageBlobTests-frameworks-${CONFIGURATION}-input-files.xcfilelist",
```

Comment:
Are these new stages being added to the build steps?

Reply:
These are all changes that Cocoapods made after I ran `pod install` but I think so, because now the test targets are declared in the Podfile, and so Xcode has to link them with the Pods project.